distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc

```diff
@@ -0,0 +1,87 @@
+
+This quick tutorial will enable install, configuring and using a basic instance of ${branding}.
+Use this setup tutorial to test or try out the features of ${branding}. For a more thorough, secure installation, see <<_installing_${branding-lowercase},Installing ${branding}>>
+
+These steps will demonstrate:
+
+- [*] <<_${branding-lowercase}_quick_install_prerequisites,Prerequisites>>
+- [*] <<_quick_install_of_${branding-lowercase},Quick Install of ${branding}>>
+- [*] <<_ingesting_sample_data,Ingesting Data>>
+
+=== ${branding} Quick Install Prerequisites
+
+These are the basic requirements to set up the environment to run a ${branding}.
+
+* Install/Upgrade to Java 8 http://www.oracle.com/technetwork/java/javase/downloads/index.html[J2SE 8 SDK]
+** The recommended version is http://www.oracle.com/technetwork/java/javase/8u60-relnotes-2620227.html[8u60] or later.
```

Comment:
I guess my point was that we probably don't want to keep having to update this version every time a new update we care about comes out. Which is why I thought saying something like `The most recent version of Java 8` or something similar.